code/game/machinery/doors/poddoor.dm

```diff
@@ -283,22 +283,40 @@
 		/obj/structure/machinery/door/airlock,
 	)
 
+/obj/structure/machinery/door/poddoor/almayer/Initialize()
+	. = ..()
+	addtimer(CALLBACK(src, TYPE_PROC_REF(/atom, relativewall_neighbours)), 10)
```

Comment:
Can we not use LateInitialize instead?

Reply:
I don't actually know how this got here, I don't recall putting it in.